mysql-test/suite/rpl/t/rpl_mts_relay_log_recovery_on_error.test

```diff
@@ -39,11 +39,11 @@
 --source include/have_binlog_format_statement.inc
 # Test requires master-info-repository=TABLE, relay-log-info-repository=TABLE
 --source include/have_slave_repository_type_table.inc
---source include/only_mts_slave_parallel_workers.inc
 --source include/only_mts_slave_parallel_type_database.inc
```

Comment:
Should this one be moved after `rpl_connection_slave.inc` as well?

Reply:
fixed